e2e-environment/synapse/auth/admin_user_auth_provider.py

```diff
@@ -2,54 +2,54 @@
 import logging
 from json import JSONDecodeError
 from pathlib import Path
-
-from twisted.internet import defer
+from typing import Any
 
 
 class AdminUserAuthProvider:
     __version__ = "0.1"
 
-    def __init__(self, config, account_handler):
+    def __init__(self, config, account_handler) -> None:  # type: ignore
```

Comment:
What is the purpose of adding the return type but then ignore type checks for this line? :thinking:

Reply:
I also just copied this code from the RSB (raiden service bundle), I guess Ulo didn#t want to spend more time ...

Reply:
Ah okay, I wasn't aware about that. So it's just fine.

PS: Another hint that these two environments should attempt to share some logic... :thinking: